front/tests/global-setup.ts

```diff
@@ -0,0 +1,54 @@
+import { test as setup } from '@playwright/test';
+import fs from 'fs';
+import type { ProjectCreateForm, RailjsonFile, StudyCreateForm } from 'common/api/osrdEditoastApi';
+import { getApiRequest, postApiRequest } from './assets/utils';
+import project from './assets/operationStudies/project.json';
+import study from './assets/operationStudies/study.json';
+import scenario from './assets/operationStudies/scenario.json';
+
+async function createDataForTests() {
+  const smallInfraRailjson: RailjsonFile = JSON.parse(
+    fs.readFileSync('../tests/data/infras/small_infra/infra.json', 'utf8')
+  );
+
+  const rollingStockJson = JSON.parse(
+    fs.readFileSync('../tests/data/rolling_stocks/electric_rolling_stock.json', 'utf8')
+  );
+
+  const createdInfra = await postApiRequest(
+    `/api/infra/railjson/`,
+    {
+      ...smallInfraRailjson,
+    },
+    {
+      name: 'small_infra_test_e2e',
+      generate_data: true,
+    }
+  );
+
+  const smallInfra = await getApiRequest(`/api/infra/${createdInfra.infra}`);
+
+  await postApiRequest('/api/rolling_stock/', {
+    ...rollingStockJson,
+    name: 'rollingstock_1500_25000_test_e2e',
+  });
+
+  const newProjectData = await postApiRequest('/api/projects/', {
```

Comment:
```suggestion
  const project = await postApiRequest('/api/projects/', {
```

Reply:
`project` is already declared when importing the json file, I can rename `newProjectData` by `Project` but in this case I have to rename the import, what do you think is better?

Reply:
Yes, projectData should be the data from the json, and project the object returned by the backend :+1: